pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.suse.salt</groupId>
   <artifactId>salt-netapi-client</artifactId>
   <packaging>jar</packaging>
-  <version>0.15.1</version>
+  <version>0.16.2-SNAPSHOT</version>
```

Comment:
This file and the above (`.travis.yml`) should not be changed by your patch, I would recommend to simply rebase your branch on top of the current master and `force-push` it.